src/components/AppNavigation/AppNavigation.vue

```diff
@@ -0,0 +1,39 @@
+<!--
+ - @copyright Copyright (c) 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+<template>
+	<div id="app-navigation" :class="navigationClass">
+		<slot />
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'AppNavigation',
+	props: {
+		navigationClass: {
```

Comment:
I think the property `navigationClass` is not needed anymore due to moving this element to it's own component. You can use `:class`, directly:

```
<app-navigation :class="{loading: loading}"> ... </app-navigation>
```

Reply:
Ah yes! You're right :see_no_evil: 
@ChristophWurst

Reply:
Of course :see_no_evil: 

@korelstar could you submit a PR please? :)

Reply:
Yep, please see #296